packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,12 +10,16 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
--   [*] Image block: Replacing the media for an image set as featured prompts to update the featured image [#34666]
 
--   [**] [Gallery block] Fix crash when adding images and selecting a gallery item [#38238]
--   [***] Font size and line-height support for text-based blocks used in block-based themes [#38205]
+## 1.71.1
 
-## 1.70.0
+-   [*] Highlight text: Check if style attribute value is defined during filtering [#38670]
```

Comment:
At this point, since this fix was actually released in version `1.70.3`, I'm wondering whether we should keep this entry, add a note that it was released on a previous version, or directly do not include it 🤔.

Reply:
I checked the entry for the previous hotfix and noticed we 'double-upped' on the release notes for that i.e. `Image block - Fix height and border regression` is listed for the hotfix in `1.61.2` and then the `1.62.1` beta. That's what led me to follow the same pattern here, but I don't feel strongly. Do you think it's confusing to leave it as it is? If so, I can remove it.

Reply:
Good catch! I either don't have a strong opinion on whether to duplicate it, but if we have done it that way in the past I think it makes sense to keep the same pattern 👍 .